src/update_engine/omaha_request_action_unittest.cc

```diff
@@ -379,6 +379,38 @@ TEST(OmahaRequestActionTest, MissingFieldTest) {
   EXPECT_TRUE(response.deadline.empty());
 }
 
+TEST(OmahaRequestActionTest, ConcatUrlSlashTest) {
+  // same as MissingFieldTest above besides the URL
+  string input_response =
+      "<?xml version=\"1.0\" encoding=\"UTF-8\"?><response protocol=\"3.0\">"
```

Comment:
These strings would probably be more readable if you wrote them like `R"(<?xml version="1.0"…)"` - less escaping clutter that way.

Reply:
That is just copied from above, so either I would change all of them or nothing